source/extensions/http/cache/file_system_http_cache/cache_file_fixed_block.h

```diff
@@ -171,7 +171,7 @@ class CacheFileFixedBlock {
       uint32_t trailer_size_;
       uint64_t body_size_;
     };
-    char as_str_[1];
+    char raw_[sizeof(uint32_t) * 4 + sizeof(uint64_t)];
```

Comment:
`raw_` member var name SGTM.

name the struct and use sizeof on the structure.

The point is moot if we decide to drop the manual serialization and just use protobufs for this.

Reply:
Removed as part of serializing differently.